.vib/wordpress-nginx/goss/vars.yaml

```diff
@@ -0,0 +1,44 @@
+binaries:
+  - nginx
+  - gosu
+  - mariadb
+  - php
+  - render-template
+  - wp
+files:
+  - paths:
+    - /opt/bitnami/nginx/conf/bitnami/php-fpm.conf
```

Comment:
As you already did with `drupal`, I would create a generic `check-nginx-libphp.yaml` that check things related to NGINX-based assets that use PHP (and thus will include [this logic](https://github.com/bitnami/containers/blob/main/bitnami/drupal-nginx/10/debian-11/rootfs/opt/bitnami/scripts/nginx-php-fpm/postunpack.sh)). WDYT?